docs/features.md

```diff
@@ -138,7 +138,7 @@ Provided by: `hls-call-hierarchy-plugin`
 
 Shows ingoing and outgoing calls for a function.
 
-![Call Hierarchy in VSCode](https://github.com/haskell/haskell-language-server/raw/2857eeece0398e1cd4b2ffb6069b05c4d2308b39/plugins/hls-call-hierarchy-plugin/call-hierarchy-in-vscode.gif)
+![Call Hierarchy in VSCode](../plugins/hls-call-hierarchy-plugin/call-hierarchy-in-vscode.gif)
```

Comment:
Didn't test the doc build but most working example images refer to relative paths within the repo, so changing this one to do the same.